dwave/optimization/_model.pyx

```diff
@@ -258,6 +272,7 @@ cdef class _Graph:
     def into_file(self, file, *,
                   Py_ssize_t max_num_states = 0,
                   bool only_decision = False,
+                  object version = None,
```

Comment:
docs missing below

Reply:
I added a short docstring. Once we expose `DEFAULT_SERIALIZATION_VERSION` I'll expand it to reference it.